.vscode/settings.json

```diff
@@ -1,6 +1,38 @@
 {
     "files.associations": {
-        "**/eng/pipelines/*.yml": "azure-pipelines"
+        "*.csproj": "msbuild",
+        "*.fsproj": "msbuild",
+        "*.globalconfig": "ini",
+        "*.manifest": "xml",
+        "*.nuspec": "xml",
+        "*.pkgdef": "ini",
+        "*.projitems": "msbuild",
+        "*.props": "msbuild",
+        "*.resx": "xml",
+        "*.rsp": "Powershell",
+        "*.ruleset": "xml",
+        "*.settings": "xml",
+        "*.shproj": "msbuild",
+        "*.slnf": "json",
+        "*.targets": "msbuild",
+        "*.vbproj": "msbuild",
+        "*.vsixmanifest": "xml",
+        "*.vstemplate": "xml",
+        "*.xlf": "xml",
+        "*.yml": "azure-pipelines"
```

Comment:
Do we have any yaml for other things than the build pipelines? I had restricted it to eng not wanting that to accidentally apply elsewhere.

Reply:
I checked, we do not